src/common/CMakeLists.txt

```diff
@@ -48,7 +48,9 @@ if(NOT DNNL_CPU_RUNTIME STREQUAL "THREADPOOL")
     list(REMOVE_ITEM SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/stream_threadpool.cpp")
 endif()
 
-if(NOT DNNL_EXPERIMENTAL_LOGGING)
+if(DNNL_EXPERIMENTAL_LOGGING)
+    include_directories(${CMAKE_CURRENT_LIST_DIR})
```

Comment:
This one is required to avoid changing spdlog source code, but has potential for side effects. Maybe it worth moving whole spdlog component to a separate folder like `thirdparty`.